.github/workflows/close-stale-prs.yml

```diff
@@ -0,0 +1,20 @@
+name: Close stale PRs
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          days-before-close: 0
+          days-before-stale: 14
```

Comment:
This seems pretty aggressive.

Reply:
I've increased `days-before-stale` to `60` and set `days-before-close` to `7`.